solr/core/src/resources/EnvToSyspropMappings.properties

```diff
@@ -0,0 +1,60 @@
+# Mapping from Environment variable to system property
+# This file only contains non-standard mappings that do not follow the standard naming convention
+# Map to nothing to avoid setting any system property for the env.variable
+SOLR_DELETE_UNKNOWN_CORES=solr.deleteUnknownCore
+SOLR_ENABLE_REMOTE_STREAMING=solr.enableRemoteStreaming
+SOLR_ENABLE_STREAM_BODY=solr.enableStreamBody"
+SOLR_HADOOP_CREDENTIAL_PROVIDER_PATH=hadoop.security.credential.provider.path
+SOLR_HOME=solr.solr.home
+SOLR_HOST=host
+SOLR_LOGS_DIR=solr.log.dir
+SOLR_PORT=jetty.port
+SOLR_TIMEZONE=user.timezone
+SOLR_TIP=solr.install.dir
+SOLR_TIP_SYM=solr.install.symDir
+SOLR_WAIT_FOR_ZK=waitForZk
+
+# These should not be mapped to system properties
+SOLR_ADDL_ARGS=
+SOLR_AUTHENTICATION_CLIENT_BUILDER=
+SOLR_HEAP=
+SOLR_HEAP_DUMP=
+SOLR_HEAP_DUMP_DIR=
+SOLR_INCLUDE=
+SOLR_JAVA_MEM=
+SOLR_JAVA_STACK_SIZE=
+SOLR_JETTY_CONFIG=
+SOLR_OPTS=
+SOLR_OPTS_INTERNAL=
+SOLR_REQUESTLOG_ENABLED=
+SOLR_SERVER_DIR=
+SOLR_SSL_CLIENT_KEY_STORE=
+SOLR_SSL_CLIENT_KEY_STORE_PASSWORD=
+SOLR_SSL_CLIENT_KEY_STORE_TYPE=
+SOLR_SSL_CLIENT_TRUST_STORE=
+SOLR_SSL_CLIENT_TRUST_STORE_PASSWORD=
+SOLR_SSL_CLIENT_TRUST_STORE_TYPE=
+SOLR_SSL_KEY_STORE=
+SOLR_SSL_KEY_STORE_PASSWORD=
+SOLR_SSL_KEY_STORE_TYPE=
+SOLR_SSL_NEED_CLIENT_AUTH=
+SOLR_SSL_OPTS=
+SOLR_SSL_TRUST_STORE=
+SOLR_SSL_TRUST_STORE_PASSWORD=
+SOLR_SSL_TRUST_STORE_TYPE=
+SOLR_SSL_WANT_CLIENT_AUTH=
+SOLR_START_WAIT=
+SOLR_STOP_WAIT=
+
+# Parking lot
+# TODO: These seemed to cause issues with the BATS ssl tests...
```

Comment:
I think these are consumed by Jetty, so probably not very useful to remove them from bin/solr, since Jetty probably need these before our app is even started?